hammer/library/jiraoperations.py

```diff
@@ -125,8 +118,9 @@ def __init__(self, config):
         self.server = self.config.jira.server
         # JIRA established session
         self.session = None
+        self.module_jira_enabled = getattr(config, module).jira if hasattr(config, module) else True
```

Comment:
Will this work if slack or jira isn't set on module config?
I see this https://github.com/dowjones/hammer/blob/dev/hammer/library/config.py#L384
This can be an issue if someone upgrades the code but don't touch configs.

Reply:
Such case handled by hasattr which under the hood intercept exception from getattr

Reply:
getattr looks for module in config. I'm talking about presence of `jira` and `slack` options in module config itself.

Reply:
New commit has been added